README.md

```diff
@@ -16,11 +16,13 @@ There are two possible ways to use the tool:
 Open `Refactoring Experiments` directory and execute the following command
 
 ```
-./gradlew runRefactoringsExperiments -Prunner=RefactoringsExperiments -PprojectsFilePath=/path/to/file.txt -PgeneratePositiveSamples
+./gradlew runRefactoringsExperiments -Prunner=RefactoringsExperiments -PprojectsDirPath=/absolute/path/to/projects/parent/folder -PdatasetsDirPath=/absolute/path/to/output/folder/ -PgeneratePositiveSamples
```

Comment:
`/absolute/path/to/projects/parent/folder` is too long. let's use  `/path/to/projects/` and `/path/to/output/`
 i suggest explaining that the paths should be absolute before the command example

Reply:
Resolved that